xbmc/addons/kodi-addon-dev-kit/CMakeLists.txt

```diff
@@ -2,6 +2,17 @@ project(kodi-addon-dev-kit)
 
 cmake_minimum_required(VERSION 2.8.7)
 
+# add a target to generate API documentation with Doxygen
+find_package(Doxygen)
+
+if(DOXYGEN_FOUND)
+  add_custom_target(doc
+    ${DOXYGEN_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/doxygen/Doxyfile
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    COMMENT "Generating API documentation with Doxygen" VERBATIM
+  )
+endif(DOXYGEN_FOUND)
+
```

Comment:
How is the current documentation of Python on http://mirrors.xbmc.org/docs/python-docs/16.x-jarvis/ generated and uploaded? From hand or somewhere automatic?

Also another question. For what is the on https://github.com/xbmc/xbmc/blob/master/codegenerator.mk#L57-L59 generated doxygen xml needed, is it to use on python itself to have some docs for Kodi? Or is it another way as source for `mirrors.xbmc.org`?